net-mail/offlineimap/offlineimap-7.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Normally you need only one version of this.
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python2_7 )
```

Comment:
Does it support any newer Pythons? This is a bit controversial given 2.7 is on its way out.

Reply:
Unfortunately it does not :( - OfflineIMAP seems to be still actively maintained, but it doesn't look like Python 3+ support is coming any time soon. 

The author started a Python 3+ compatible rewrite ([IMAPfw](https://github.com/OfflineIMAP/IMAPfw)) but it hasn't seen a commit in 3 years.

Reply:
Bleh. It may be that you need to remove the support eventually.